packages/frontend/src/components/dialogs/QrCode.tsx

```diff
@@ -235,6 +239,63 @@ export function QrCodeScanQrInner({
     [accountId, processQr, onDone, openAlertDialog]
   )
 
+  // General handler for scanning results coming from the "jsqr" library.
+  //
+  // Additionally we have checks in place to make sure we're not firing any
+  // callbacks when this React component has already been unmounted.
+  const handleScanResult = useCallback(
+    result => {
+      let unmounted = false
+
+      if (unmounted) {
+        return
+      }
+
+      handleScan(result.data)
+
+      return () => {
+        unmounted = true
+      }
```

Comment:
So this is copy-pasted. Because `unmounted` doesn't appear to do anything. Just saying.

Reply:
That was code that already existed. I removed it since I could not create a case, where the callback was called in an unmounted component, even with clicking "through" the overlay to close the dialog immediately
@Simon-Laux do you remember why that was added? 
The comment said:
>  // Additionally we have checks in place to make sure we're not firing any
 > // callbacks when this React component has already been unmounted.

Reply:
If you consider this change to "risky" we could either readd it or postpone this PR for the current release

Reply:
To me this looks like dead code. `unmounted` is never read after initialization, it's a new variable on each function execution, and it's only scoped to the function.